File: 8_kyu/No_zeros_for_heros.java
public class NoBoring {
        public static int noBoringZeros(int n) {
            
            if (n == 0) {
              return 0;
            }else{
              return n % 10 != 0 ? n : noBoringZeros(n / 10);          
            } 
    
        }
    }

File: 8_kyu/No_zeros_for_heros.js
  const noBoringZeros = n => {
    
    if (n === 0) {
      return 0;
    }
    
    n = n.toString().split('').reverse();
  
    // Eliminar ceros al principio del array
    while (n[0] === '0') {
      n.shift();
    }
  
    return +(n.reverse().join(''));
  }